libexec/tfenv-uninstall

```diff
@@ -7,21 +7,14 @@ source ${TFENV_ROOT}/libexec/helpers
 
 declare version_requested version regex
 
-if [ -z "${1}" ]; then
-  version_file="$(tfenv-version-file)"
-  if [ "${version_file}" != "${TFENV_ROOT}/version" ];then
-    version_requested="$(cat ${version_file} || true)"
-  fi
-else
-  version_requested="${1}"
-fi
+get_version_requested "${1}"
```

Comment:
I'm not sure it's a good idea to invoke a common function that uses tfenv-min-required in `uninstall`

Reply:
> Proposing to close this since #111 should provide the resolution
It seems to be a good idea.